ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/networking/libp2p/rpc/GoodbyeMessage.java

```diff
@@ -33,6 +33,9 @@
   public static final UnsignedLong REASON_FAULT_ERROR = UnsignedLong.valueOf(3);
   public static final UnsignedLong MIN_CUSTOM_REASON_CODE = UnsignedLong.valueOf(128);
 
+  // Custom reasons
+  public static final UnsignedLong REASON_UNABLE_TO_VERIFY_NETWORK = UnsignedLong.valueOf(128);
```

Comment:
Not sure if we should be using custom codes at this point, but the defaults are not very expressive.